.vsts-ci.yml

```diff
@@ -99,8 +100,6 @@ extends:
               - name: _OfficialBuildArgs
                 value: ''
             steps:
-            - checkout: self
-              clean: true
```

Comment:
Did you remove clean: true because this is true by default?

Reply:
I discussed with @ mmitche offline, and according to him:
> It's because microbuild's tasks are getting installed to the source directory due to an arcade change to enable linux/mac signing. The checkout step you all run does a clean which deletes the files. 
> The thing is that the checkout step is not necessary since it happens by default IIRC